src/behaviors/utils.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import type { Document, Element } from 'hyperview/src/types';
+import { shallowCloneToRoot } from 'hyperview/src/services';
+
+/**
+ * The elements in the Document with the given ids will be set to
+ * visible or hidden based on showIndicators.
+ * Returns a new Document object with the toggled indicators.
+ */
+export const toggleIndicators = (
+  ids: Array<string>,
+  showIndicators: boolean,
+  root: Document,
+): Document => {
+  let newRoot = root;
+  ids.forEach(id => {
```

Comment:
(not a blocker) As a general reco for new code added, I'd suggest to try avoid variables mutations when possible, i.e.

```js
return ids.reduce((newRoot, id) => {
  const indicatorElement: ?Element = root.getElementById(id);
	if (!indicatorElement) {
    return root;
  }
  indicatorElement.setAttribute('hide', showIndicators ? 'false' : 'true');
	return shallowCloneToRoot(indicatorElement);
}, root);
```